### ISSUE TITLE
DS Hue App DP are not writable

### ISSUE THREAD

**Issue by Draygv2** [ioBroker/ioBroker.digitalstrom#241]
**Describe the bug**  
If I change the value of level.brightness it change to red. The lampe doesn't make anything. Same for colortenp. If I change a preset it work instantly

**To Reproduce**  
1. Add Hue App in DS and load all lamps
2. open objects in iob and go to ds/device/hue/lamp xy
3. change the value
4. See number in red and no effect on the lamp

**Expected behavior**  
The lamp shine in the level I put in.

**Screenshots & Logfiles**  
There is no issue in the protocol on debug mode

**Versions:**  
 - Adapter version: <2.3.0>
 - JS-Controller version: <4.0.23>
 - Node version: <v14.20.0> 
 - Operating system: <ubuntu>

**Comment by Apollon77** [comment]
Can you see or set the values in the DS web ui? Frommmy knowledge not. And it is not documents in any API spec on how to set vaues using DS APIs ... So no idea how to do it.
Maybe the new "get values API" delivers the values but thats not integrated yet. 

When you have ioBroker I also think that using te hue adapzter is much more powerful than using DS (which is controlling hue then), or?!

**Comment by Draygv2** [comment]
Yes I can change it on the "Aktivitäten" tab.
Yes you are right it is much easier with the hue adapter. I will do it so.

**Comment by Apollon77** [comment]
Can you post a screenshot of this "Aktivitäten" Tab? Can you iopen the browser Develoepr tools Tab Network and check which requests are done there if you change it??

**Comment by Draygv2** [comment]
![image](https://user-images.githubusercontent.com/62559741/187019669-8fdba84c-763d-4b1d-b430-cd0b0d6b1105.png)

Network request:
Anforderungs-URL: https://10.2.2.60/json/device/setOutputChannelValue?_dc=1661585460008&channelvalues=brightness%3D62&applyNow=true&dsuid=718e93da07975bd780aad8a5b67b933500
Anforderungsmethode: GET
Statuscode: 200 OK
Remoteadresse: 10.2.2.60:443
Referrer-Richtlinie: strict-origin-when-cross-origin

I hope that is what you looking for

**Comment by Apollon77** [comment]
Yes kind of this ...  ut NOT for brightness :-) Briggthness is known and works :-)

if you change something there should be also POST calls

**Comment by Draygv2** [comment]
But I have the problem with brightness. Nothing happens if I put in a value between 0 and 100. The colortemp is also not working.
I have just GET in the networkanalyse. Do I get it somewhere else?
![grafik](https://user-images.githubusercontent.com/62559741/188810783-15e1d42d-bb3f-4104-a99d-39a70ad3644f.png)


Here the request for colortemp:
{
	"GET": {
		"scheme": "https",
		"host": "10.2.2.60",
		"filename": "/json/device/setOutputChannelValue",
		"query": {
			"_dc": "1662533843874",
			"channelvalues": "colortemp=370",
			"applyNow": "true",
			"dsuid": "aa47d858fb7a564480c44dac05015ccd00"
		},
		"remote": {
			"Adresse": "10.2.2.60:443"
		}
	}
}

{
	"Antwortkopfzeilen (331 B)": {
		"headers": [
			{
				"name": "Content-Length",
				"value": "11"
			},
			{
				"name": "Content-Type",
				"value": "application/json; charset=utf-8"
			},
			{
				"name": "Date",
				"value": "Wed, 07 Sep 2022 06:57:24 GMT"
			},
			{
				"name": "Server",
				"value": "lighttpd/1.4.48"
			},
			{
				"name": "Set-Cookie",
				"value": "csrf-token=; Path=/"
			},
			{
				"name": "Strict-Transport-Security",
				"value": "max-age=0"
			},
			{
				"name": "X-Content-Type-Options",
				"value": "nosniff"
			},
			{
				"name": "X-Frame-Options",
				"value": "sameorigin"
			}
		]
	}
}
